backend/src/users.spec.js

```diff
@@ -202,6 +200,7 @@ describe("mutations", () => {
                 errors: [expect.objectContaining({ message: "No user with this email" })],
             });
         });
+
         it("throws error if the password is incorrect", async () => {
             await expect(loginAction("jonas@jonas.com", "wrongPassword")).resolves.toMatchObject({
                 data: {
```

Comment:
Data created in mutations are persisted and your queries and mutations still work as expected.

Reply:
+1